src/psbt/mod.rs

```diff
@@ -279,6 +280,37 @@ impl<'psbt, Pk: MiniscriptKey + ToPublicKey> Satisfier<Pk> for PsbtInputSatisfie
             <Satisfier<Pk>>::check_older(&Older(seq), n)
         }
     }
+
+    fn lookup_hash160(&self, h: hash160::Hash) -> Option<[u8; 32]> {
```

Comment:
In fact I think we need an introduction of a preimage type like in https://github.com/LNP-BP/rust-lnpbp/blob/master/src/bp/hlc.rs#L25-L60 — but I assume this is outside of the scope of the present PR

Reply:
It's a pity we do not have a const generics with our MSRV, otherwise we can generalize the preimage type.

Reply:
Yeah, right now these function signatures are decided by the satisfier Trait. We could have a new type for PreImage.